### ISSUE TITLE
metamask shows higher tx fee due to gas limit, any better way to show the actual fee?

### ISSUE THREAD

**Issue by gupadhyaya** [harmony-one/ethhmy-bridge.frontend#28]
1. metamask calculates the amount based on how much user is willing to spend (based on gas limit), and it is not exactly the amount which will be charged for the user. Most users think that bridge is going to cost them $19 in the below picture. Any way to configure metamask to not scare people? (the proficient metamask users may understand this, but not many people understand this).

<img width="1600" alt="Screen Shot 2020-10-10 at 8 41 41 AM" src="https://user-images.githubusercontent.com/13854340/95659337-2c270a80-0ad5-11eb-8ac8-492a7005cb8b.png">

2. for the approve transaction, metamask shows fee in dollars, however for the lock, it shows in ether. can we have this also show in dollars? or make the first one in ether as well. 

<img width="1654" alt="Screen Shot 2020-10-10 at 8 43 15 AM" src="https://user-images.githubusercontent.com/13854340/95659333-27faed00-0ad5-11eb-8150-02130d6b70cc.png">

basically, figure out a consistent way to show the tx fee (either dollars or ether).
